matchzoo/models/mvlstm_model.py

```diff
@@ -0,0 +1,64 @@
+"""An implementation of MVLSTM Model."""
+import typing
+
+import keras
+import keras.backend as K
+
+from matchzoo import engine
+
+
+class MVLSTMModel(engine.BaseModel):
+    """
+    MVLSTM Model.
+    Examples:
+        >>> model = MVLSTMModel()
+        >>> model.guess_and_fill_missing_params()
+        >>> model.build()
+    """
+
+    @classmethod
+    def get_default_params(cls) -> engine.ParamTable:
+        """:return: model default parameters."""
+        params = super().get_default_params(with_embedding=True)
+        params['optimizer'] = 'adam'
+        params['input_shapes'] = [(5,), (300,)]
+        params['embedding_output_dim'] =  50
+        params.add(engine.Param('trainable_embedding', False))
```

Comment:
`super().get_default_params(with_embedding=True)` this already gives the trainable option.